lib/logstash/outputs/email.rb

```diff
@@ -4,6 +4,24 @@
 
 # Send email when an output is received. Alternatively, you may include or
 # exclude the email output execution using conditionals. 
+#
+# ### Usage Example
```

Comment:
Please use `==== Usage Example` instead of `###`. We are using asciidoc instead of markdown